### PR TITLE
project+community+exp added; css

### PR DESCRIPTION
**Pull request by pierreetiennealary**
button added
images directory added
css directory added
experimental-organization (project + experiments) and community .md updated

### CODE REVIEW

File: docs/community.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
 # Community
 
+## Persons
+Phis community is composed of persons which can be either:
+- **persons** identified by their names, email and affiliation, that Phis projects and expertiments can refer too
```
Comment (annetireau) [inline docs/community.md]: Only email and affiliation are mandatory

File: docs/community.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+- persons that own a Phis account, and are refered to as **users**
+
+## Access rights
+
+### User account levels of access
+`Guest`: a guest (or "simple") user gets access to public data and data accessible to the groups the user belongs to.
```
Comment (annetireau) [inline docs/community.md]: User access rights depend of users groups.

Maybe explain Group in first or just distinct admin user and classic user.
An admin is usually the platform or installation manager and data managers.

File: docs/community.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+
+`Owner`: on top of the guest user access rights, an owner can create or delete user accounts.
+An owner can also create groups and assign users to those groups.
+There is generally no more than one or two owner per platform.
+
+### Group restrictions
+A given information can be visible to specific groups. A Phis user not belonging to those groups will not be able to view the data linked to them.
+The list of concepts that can be assign to groups is:
```
Comment (annetireau) [inline docs/community.md]: not really clear...

Currently, access rights are manage by experiment.

File: docs/community.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+
+- experiments
+- ...
+
+### Public data
+An information assigned to no group is by default set as public. For instance, an experiment not belonging to any group will be viewable to every Phis user, e.g. to every person owning a Pḧis account.
```
Comment (annetireau) [inline docs/community.md]: Maybe add that normally research data should become public data after several years.
//we should confirm and specify with Pascal)

File: docs/community.md [hunk 5]
```diff
@@ -9,9 +39,11 @@ accounts and authorization access
 
 `Tools > Persons`
 
-Open Researcher and Contributor ID (ORCID)
+`ORCID`. Open Researcher and Contributor ID. Optionnal.
+
```
Comment (annetireau) [inline docs/community.md]: You can create and get your orcid at https://orcid.org/

File: docs/experimental-organization.md [hunk 6]
```diff
@@ -3,38 +3,163 @@
 ## Projects
 
 ### What are projects ?
+A project is the higher level of organisation in PHIS.
```
Comment (annetireau) [inline docs/experimental-organization.md]: Maybe add the relation with contract...

File: docs/experimental-organization.md [hunk 7]
```diff
@@ -3,38 +3,163 @@
...
+A single experimentation can include several experiments from various experimental campaigns.
```
Comment (annetireau) [inline docs/experimental-organization.md]: not really clear form me
~~ maybe something like : A project can hold several experiments from various experimental campaigns.
And many projects can articipate to an experimentation.

File: docs/experimental-organization.md [hunk 8]
```diff
@@ -3,38 +3,163 @@
...
 
 ### What are project properties ?
+Projects fact sheets are all public, i.e. visible for every PHIS user, including those unrelated to them.
+However, experiments and objects included in a project are accessible only to the users defined as owner or guest of this project.
```
Comment (annetireau) [inline docs/experimental-organization.md]: Rights are on experiments, not project.
Maybe start of proposal : However, experimental data included in a project are accessible only to users who belong to an owner or guest groups of the experimentation.

File: docs/experimental-organization.md [hunk 9]
```diff
@@ -3,38 +3,163 @@
...
+The mandatory fields, followed by an asterisk, are:
+- acronyme
+- name
+- date (start / end)
+
+However, it is highly recommended to fill every field. If possible, fields requiring plain text (keywords, description) should be answered to in english.
+
+`URI`. The project URI is automatically created using the acronym provided by the user.
+
+`Acronym`. Short and meaningful, the acronym will be the preferred way to refer to the created project.
```
Comment (annetireau) [inline docs/experimental-organization.md]: Usually the acronym of the research project

**Review by annetireau — COMMENTED** [review]
First comments

File: docs/community.md (hunk 1 above)
Comment (pierreetiennealary) [reply docs/community.md]: ok, I've corrected it (cf. pull request that will follow today)

File: docs/experimental-organization.md (hunk 7 above)
Comment (pierreetiennealary) [reply docs/experimental-organization.md]: ok, I typed "single experimentation" instead of "signle project"

**Pull request closed without merging**